projects/packages/sync/src/modules/class-woocommerce.php

```diff
@@ -257,34 +257,6 @@ public function action_woocommerce_remove_order_items( WC_Order $order, $type )
 		}
 	}
 
-	/**
-	 * Expand order item IDs to order items and their meta.
-	 *
-	 * @access public
-	 *
-	 * @todo Refactor table name to use a $wpdb->prepare placeholder.
-	 *
-	 * @param array $args The hook arguments.
-	 * @return array $args Expanded order items with meta.
-	 */
-	public function expand_order_item_ids( $args ) {
```

Comment:
Is there a reason for renaming `expand_order_item_ids`? Since this is a public method we can't remove it before at least deprecating it :)

Reply:
Good catch, I'll deprecate it then. We are not really expanding them anymore in the `jetpack_sync_before_send_jetpack_full_sync_*` hook, but just building the desired structure by wpcom.

Also, ideally `build_full_sync_action_object` will be in the parent class. Probably in a future PR when more modules use the same structure.

Reply:
Done in [2c84768](https://github.com/Automattic/jetpack/pull/41433/commits/2c84768be408891de02295c25368d12c6ee67273)